docs/json_rpc_api.mediawiki

```diff
@@ -322,6 +322,10 @@ the method name for further details such as parameter and return information.
 |Y
 |Returns information about an unspent transaction output.
 |-
+|[[#gettxoutsetinfo|gettxoutsetinfo]]
+|Y
```

Comment:
Even though this should ideally be available to a limited user, it is currently incredibly expensive to use, so I don't think it should be available to the limited user.

Also, it's not in the limited user map, so this doesn't match reality currently.